cmd/rancher-desktop-guestagent/main.go

```diff
@@ -150,8 +152,16 @@ func main() {
 
 		if *enableKubernetes {
 			group.Go(func() error {
+				k8sServiceListenerIP := net.ParseIP(*k8sServiceListenerAddr)
+
+				if k8sServiceListenerIP == nil || !(k8sServiceListenerIP.Equal(net.IPv4zero) ||
+					k8sServiceListenerIP.Equal(net.IPv4(127, 0, 0, 1))) { //nolint:gomnd
```

Comment:
Can you also include an additional `nolint` comment format, so that in the future we can determine why we are using `nolint` just by reading the comment. The format is like:
```
// nolint:gosec // wsl-distro release URL
```
you can find an example [here](https://github.com/rancher-sandbox/rancher-desktop/blob/c07918276a3e4399dd4fea68a93cce2c8fcc6954/src/go/vtunnel/test/e2e/connectivity_test.go#L391)